src/System.CodeDom/src/System/CodeDom/CodeAttributeArgumentCollection.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+
+namespace System.CodeDom
+{
+    [Serializable]
+    public class CodeAttributeArgumentCollection : CollectionBase
```

Comment:
> Should the CodeDom collections avoid inheriting from CollectionBase, to avoid a public API dependency on the non-generic collections?

No, we need to leave this as is.

> For example, X509CertificateCollection inherits from CollectionBase in the full framework, but in CoreFX it only inherits from the IList interface.

If we removed that base class, we need to put it back. cc: @bartonjs